_data/saks1209am/saksham.jpg

```diff
@@ -0,0 +1 @@
+![Saksham](https://user-images.githubusercontent.com/30996074/82549799-2149f880-9b7b-11ea-85dc-959722465e91.JPG)
```

Comment:
```suggestion
Put the whole file name as `Saksham.jpg`
```

No you don't have to pass any url. Just add the image in the folder and commit.

Reply:
Simply delete this img file and paste a original `Saksham.jpg` file which is your picture.

Reply:
Did it bro. Do i again need to create a new PR?